### PR TITLE
Adding API proxy endpoint, skipping NYI tests

### PR DESCRIPTION
**Pull request by JayneTosh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "food-api-tests",
   "version": "1.0.0",
   "description": "BBC Food API tests",
-  "author": "Jack Stevenson",
+  "author": "Jack Stevenson, Jayne Tosh",
```
Comment (VictoriaSheppard) [inline package.json]: disapprove

Comment (VictoriaSheppard) [reply package.json]: glory hoggers

Comment (JayneTosh) [reply package.json]: No

**Comment by JayneTosh** [comment]
Closing since we're using a different approach

**Pull request closed without merging**